common/helpers/field.js

```diff
@@ -177,14 +179,59 @@ function insertItemConditional({ key, field }) {
   }
 }
 
+async function mapAssessmentQuestion([key, field]) {
+  if (!Object.prototype.hasOwnProperty.call(field, 'items')) {
+    return [key, field]
+  }
+
+  const assessmentQuestions = await referenceDataService.getAssessmentQuestions(
+    key
+  )
+
+  field.items = assessmentQuestions
+    .filter(referenceDataHelpers.filterDisabled())
+    .map(mapAssessmentQuestionToConditionalField)
+    .map(mapAssessmentQuestionToTranslation)
+    .map(mapReferenceDataToOption)
+
+  return [key, field]
+}
+
+function mapAssessmentValidation([key, field], index, obj) {
+  if (!isNull(field.dependent)) {
+    return [key, field]
+  }
+
+  const parentField = Object.values(fromPairs(obj)).find(field =>
```

Comment:
Wonder if using lodash for this would make it a bit easier to read? We're already using it in other places so don't mind using it for more complicated finds.

Reply:
have removed the need for this code